modules/auxiliary/scanner/smb/smb_ms17_010.rb

```diff
@@ -167,7 +168,7 @@ def run_host(ip)
     rescue ::Rex::Proto::SMB::Exceptions::LoginError
       print_error("An SMB Login Error occurred while connecting to the IPC$ tree.")
     rescue ::Exception => e
-      vprint_error("#{e.class}: #{e.message}")
+      print_error("#{e.class}: #{e.message}")
```

Comment:
I assume this debugging code can be removed now? 👀

Reply:
No, I made this a permanent fix since we'd want users to be aware of the specific error being thrown when running `check` and when running the module on it's own.